### PR TITLE
XW-41 | Video thumbnails

### PR DESCRIPTION
**Pull request by hakubo**

### CODE REVIEW

File: front/main/app/templates/components/article-media-gallery.hbs [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-			forceHeight=imageSize
-			cropMode=cropMode
-		}}
-	</span>
+	{{article-media-image
+		url=item.url
+		link=item.link
+		ref=ref
+		galleryRef=item.galleryRef
+		itemType=(concat 'gallery-' item.type)
```
Comment (rogatty) [inline front/main/app/templates/components/article-media-gallery.hbs]: `itemType` is passed here as an attribute and then used as a computed property. I don't think we should do that.

Comment (hakubo) [inline front/main/app/templates/components/article-media-gallery.hbs]: do you have an idea how to do that more cleanly?

Comment (hakubo) [inline front/main/app/templates/components/article-media-gallery.hbs]: or I could add another param
itemContext='article|gallery'

and use that in the component

how does that sound?

Comment (rogatty) [inline front/main/app/templates/components/article-media-gallery.hbs]: The first thing that comes to my mind is that we already know it's a gallery (`galleryRef` attribute was passed) so we can use it in the computed property.

**Comment by rogatty** [comment]
LGTM

**Comment by bognix** [comment]
LGTM

**Pull request merged**